pom.xml

```diff
@@ -76,15 +76,15 @@
   </repositories>
 
   <properties>
-    <cdap.version>4.1.0-SNAPSHOT</cdap.version>
+    <cdap.version>4.3.1</cdap.version>
     <hadoop.version>2.3.0</hadoop.version>
     <junit.version>4.11</junit.version>
     <slf4j.version>1.7.5</slf4j.version>
     <hydrator.version>1.5.0-SNAPSHOT</hydrator.version>
     <widgets.dir>widgets</widgets.dir>
     <docs.dir>docs</docs.dir>
     <app.parents>
-      system:cdap-etl-batch[4.0.0,10.0.0-SNAPSHOT),system:cdap-etl-realtime[4.0.0,10.0.0-SNAPSHOT),system:cdap-data-pipeline[4.0.0,10.0.0-SNAPSHOT),system:cdap-data-streams[4.0.0,10.0.0-SNAPSHOT)
+      system:cdap-data-pipeline[4.0.0,10.0.0-SNAPSHOT),system:cdap-data-streams[4.0.0,10.0.0-SNAPSHOT)
```

Comment:
Though it was this way before, I don't think we should have the upper limit cross a major version, since we say we may introduce backwards incompatible changes at a major version.